packages/react/src/components/Icon/index.tsx

```diff
@@ -16,6 +16,9 @@ function Icon({ label, className, type, ...other }: IconProps) {
   const [IconSVG, setIcon] = useState<React.ComponentType<any> | null>(null);
 
   useEffect(() => {
+    if (process.env.JEST_WORKER_ID !== undefined) {
```

Comment:
What if we checked `NODE_ENV` instead?

```suggestion
    if (process.env.NODE_ENV === "test") {
```